app/messages.js

```diff
@@ -1,16 +1,13 @@
-define(['jquery', 'app/timer', 'app/string'], function ($, timer, string) {
+define(['jquery', 'app/lang', 'app/timer', 'app/string'], function ($, lang, timer, string) {
   /* Function show for displaying normal, danger and warning messages.
   * The message is included as html and a class with the corresponding type is
   * added. Within the switch statement the icon is added.
   */
-  
-  const THOUSAND = 1000;
 
   return {
     show: function(message, type, countdown) {
-      countdown /= THOUSAND;
       if (countdown > 0) {
-        $("#show .message").html(message + "<br> <span class=\"countdownnotice\">Press enter to continue or wait <span class=\"countdown\" data-seconds=\"" + countdown + "\"></span> second" + string.pluralIfAppropriate(countdown) + ".</span>");
+        $("#show .message").html(message + "<br> <span class=\"countdownnotice\">" + lang("message_enterorwait", "<span class=\"countdown\" data-seconds=\"" + countdown + "\"></span>"));
```

Comment:
Nice work! I'll merge this sprint into the master later day. 

Sent from my phone

> On 18 May 2016, at 15:03, sdejong <notifications@github.com> wrote:
> 
> In app/messages.js:
> 
> >        if (countdown > 0) {
> > -        $("#show .message").html(message + "<br> <span class=\"countdownnotice\">Press enter to continue or wait <span class=\"countdown\" data-seconds=\"" + countdown + "\"></span> second" + string.pluralIfAppropriate(countdown) + ".</span>");
> > +        $("#show .message").html(message + "<br> <span class=\"countdownnotice\">" + lang("message_enterorwait", "<span class=\"countdown\" data-seconds=\"" + countdown + "\"></span>"));
> Yes for all three cases. All comments have now been addressed.
> 
> —
> You are receiving this because you were assigned.
> Reply to this email directly or view it on GitHub
>